spec/features/articles_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+describe ArticlesController, type: :feature do
+  let(:user) { create(:user) }
+  before { login user }
+
+  describe '#index GET /articles' do
+    describe 'sort by shipped_at desc' do
+      let!(:today) { create(:article, :with_tags, shipped_at: Date.today) }
```

Comment:
__[RuboCop]__

Rails/Date: Do not use `Date.today` without zone. Use `Time.zone.today` instead.


<!-- Comment by [SideCI](https://sideci.com) -->